lampod-cli/src/main.rs

```diff
@@ -44,15 +48,57 @@ fn main() -> error::Result<()> {
     Ok(())
 }
 
+fn write_words_to_file<P: AsRef<Path>>(path: P, words: &Option<String>) -> io::Result<()> {
+    let mut file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(path)?;
+
+    match words {
+        Some(ref value) => {
+            file.write_all(value.as_bytes())?;
+        }
+        None => {
+            file.write_all(b"")?;
+        }
```

Comment:
I am not able to understand this, why ?

Reply:
OK let think more about this!

What is the case where the mnemonic is None?

Reply:
Yes, ususally this case wouldnt happen. This will only happen if the mnemonic is none, maybe the user doesnt enter anything when the mnemonic is asked.

Reply:
We should return an error if the user does not pass the correct mnemonic!

Your code is writing an invalid seed, which in some cases can lead to lost funds. This task that you are doing may looks trivial but it is crucial for user funds, so please pay attention on the validation

Reply:
Okay